api/src/main/java/io/strimzi/api/kafka/model/JvmOptions.java

```diff
@@ -68,13 +68,13 @@ public void setServer(Boolean server) {
         this.server = server;
     }
 
-    @Description("Disable garbage collection logging")
-    public boolean isGcLoggingDisabled() {
-        return gcLoggingDisabled;
+    @Description("Enable or disable Garbage Collection logging. It is enabled by default.")
```

Comment:
This description is better. But whatever you use the descriptions should be consistent. And why is Garbage Collection capitalised?
```suggestion
    @Description("Enable or disable garbage collection logging. It is enabled by default.")
```

Reply:
Because we always says GC and not gc I thought that having it capitalize was a good idea

Reply:
Why have you changed the other description to "Specifies whether the Garbage Collection logging is enabled", but left this one as "Enable or disable garbage collection logging. It is enabled by default.", surely they should be consistent?

Reply:
Fixed. I forgot about it!